### ISSUE TITLE
[2pt] Add retry to `manage replicasets` stage

### ISSUE THREAD

**Issue by opomuc** [tarantool/ansible-cartridge#142]
When there are a lot of instances or they are under some high load `manage replicasets` can fail and it will stop any pipeline.

BUT, at the same time instances ARE running and everything is ok.

Take a look at some example:

```
TASK [ansible-cartridge : Set one control instance to join replicasets] ****************************************************************************************************************
ok: [storage-AE-0 -> xxx]

TASK [ansible-cartridge : Get replicasets] *********************************************************************************************************************************************
ok: [storage-AE-0]

TASK [ansible-cartridge : Manage replicasets] ******************************************************************************************************************************************
included: .../ansible/roles/ansible-cartridge/tasks/manage_replicaset.yml for storage-AE-0

TASK [ansible-cartridge : Manage storage-AE replicaset] ********************************************************************************************************************************
fatal: [storage-AE-0 -> xxx]: FAILED! => {"changed": false, "msg": "Failed to create \"storage-AE\" replicaset: \"xxx\": Connection timed out"}

NO MORE HOSTS LEFT *********************************************************************************************************************************************************************

PLAY RECAP *****************************************************************************************************************************************************************************
storage-AE-0               : ok=12   changed=0    unreachable=0    failed=1    skipped=2    rescued=0    ignored=0   
```

I propose to at least add a retry to that stage.

**Comment by dokshina** [comment]
It seems that this issue would be solved by #165.

**Comment by hackallcode** [comment]
These tasks have been merged into one by #165. So, in the current version of the Ansible role, there is no such problem.

**Comment by dokshina** [comment]
Now it looksmlike duplicate of #169.

Moreover #169 uses new `edit_topology` approach.
